README.md

```diff
@@ -124,7 +121,7 @@ In the browser:
 ```html
 <!-- Loads `UserGate` into `window`. There is also a minified version available, `dist/bundle.min.js`.
 If anyone would like to produce a UMD version, a pull request would be very welcome! -->
```

Comment:
Curious, to round off our modernization, is it easy to do this with Browserify? That would mean bundling https://github.com/mixmaxhq/user-gate/blob/master/src/gate/index.js directly—we would no longer need https://github.com/mixmaxhq/user-gate/blob/master/src/gate/exports.js, since Browserify would generate wrapper code that loaded `UserGate` as a browser global _or_ assigned to `module.exports` if the latter was present (as synthesized by Rollup). This would mean that we would no longer need to call `noConflict` where we import user-gate.

Reply:
And then once you did this, you could replace this with a code sample that looked like our pre-bundled source, like the `import` example here https://developer.mixmax.com/docs/introduction-widget-sdk. It's commonly understood that modern client-side libraries require bundling before inclusion in an HTML page.

Reply:
Supposedly it is easy by passing `-s`: https://github.com/browserify/browserify#usage. I'll take a stab at it.

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/7e6e68d9ed15d54eb1e78190625e797b0cd8e5c8